drivers/SmartThings/matter-switch/fingerprints.yml

```diff
@@ -393,6 +393,16 @@ matterManufacturer:
     vendorId: 0x1168
     productId: 0x044E
     deviceProfileName: plug-binary
+  - id: "4456/1019"
+    deviceLabel: Matter Smart Filament  Bulb ST58 CCT
```

Comment:
very small nit: there is an extra space in the device label name between "Filament" and "Bulb"

Reply:
This is how the partner submitted it, and if we change it may break things down stream so unfortunately I think its best to leave this as is.